src/c++/perf_analyzer/client_backend/client_backend.h

```diff
@@ -199,7 +199,7 @@ struct InferOptions {
   explicit InferOptions(const std::string& model_name)
       : model_name_(model_name), model_version_(""), request_id_(""),
         sequence_id_(0), sequence_id_str_(""), sequence_start_(false),
-        sequence_end_(false)
+        sequence_end_(false), triton_enable_empty_final_response_(true)
```

Comment:
Is this where PA is setting the value to True? Is this class owned by PA and only used by PA?

Reply:
Yes